apis/core/lib/domain/cube-projects/create.ts

```diff
@@ -26,7 +26,7 @@ export async function createProject({
   project.addOut(rdfs.label, label)
     .addOut(dcterms.creator, user)
 
-  if (resource.out(cc.projectSourceKind).value === 'CSV') {
+  if (shape('cube-project/create#CSV').equals(resource.out(cc.projectSourceKind).term)) {
```

Comment:
It would be really great to not have strings but a variable here for `'cube-project/create#CSV'` as well :smile:

Reply:
The string also gets type-checked. Typos will raise TS errors

Reply:
Code completion and this. That's fine with me then. If I only have a restricted set of possibilities it is always nice to not have to go to the source to know what is allowed.